includes/cub3D.h

```diff
@@ -30,23 +30,26 @@ typedef struct s_rgb
 
 typedef struct s_identifiers_data
 {
-	char	*north_texture;
-	char	*south_texture;
-	char	*west_texture;
-	char	*east_texture;
+	char	*path_to_north_texture;
```

Comment:
you dont want to keep it just "north_texture"? we know its a path, and it makes the name way longer (i.e. annoying to use in the code), but lmk if you disagree